parts/k8s/kubernetesmastervarsvmss.t

```diff
@@ -0,0 +1,286 @@
+    "maxVMsPerPool": 100,
```

Comment:
How different are the variables for VMSS compared to VMAS? Would it make sense to have one common file with a block specific to VMSS to avoid having to maintain duplicate code?